### ISSUE TITLE
status enum and StatusGetList method

### ISSUE THREAD

**Issue by dwcai** [Parature/ParatureSDK#10]
It looks to me that **status** enum in ParaEnums.cs should be **Status**. When I tried to send a StatusGetList call, I am getting the following error: 
```
The URI is not valid: Invalid Entity Type: status
```

If I rename status enum to **Status** (using Refactoring), I am getting a different error message when I make the call. 
```
The operation: List for Entity Type: Status is not supported
```
Is this status entity really not meant to be readable? I do see that we may want to read from this entity in some occasions, particularly for someone who wants to work with Status programatically. 

Thanks!

**Comment by brtubb** [comment]
We are trying to move away from some enums for this reason. This API call actually needs the "status" uri to be all lower call. The correct call will look like this:

https://demo.parature.com/api/v1/<account>/<dept>/Customer/status
https://demo.parature.com/api/v1/<account>/<dept>/Csr/status
etc

I'm looking into why the SDK is failing to allow retrieval of statuses. We agree - this should be retrievable.

**Comment by brtubb** [comment]
Happy to report this is fixed in 2.0.4 on NuGet and in master. Sample usage:

```
var ticketStatus = ApiHandler.Ticket.Status.GetList(creds);
var custStatus = ApiHandler.Customer.Status.GetList(creds);
var csrStatus = ApiHandler.Csr.Status.GetList(creds);
```

Please note that the namespace was adjusted to be more intuitive, but may cause some breakage.
